frameworks/elastic/docs/install.md

```diff
@@ -129,7 +129,7 @@ Any other modifiable settings are covered by the various Elasticsearch APIs (clu
 
 Each task in the cluster performs one and only one of the following roles: master, data, ingest, coordinator.
 
-The default placement strategy specifies no constraint except that all the master nodes are distributed to different agents. You can specify further [Marathon placement constraints](http://mesosphere.github.io/marathon/docs/constraints.html) for each node type. For example, you can specify that data nodes are never colocated, or that ingest nodes are deployed on a rack with high-CPU servers.
+The default placement strategy specifies that no two master nodes are distributed to the same agent, and no two data nodes are distributed to the same agent. You can specify further [Marathon placement constraints](http://mesosphere.github.io/marathon/docs/constraints.html) for each node type. For example, you can specify that ingest nodes are deployed on a rack with high-CPU servers.
```

Comment:
All node types are configured with unique hostname now (master, data, coordinator (sic) and ingest).